packages/components/src/components/checkbox/checkbox.test.ts

```diff
@@ -320,4 +320,22 @@ describe('<sd-checkbox>', () => {
       expect(indeterminateIcon).to.be.null;
     });
   });
+
+  describe('should contain aria-invalid', () => {
+    it('is false when required and unchecked', async () => {
+      const el = await fixture<HTMLFormElement>(html` <sd-checkbox required></sd-checkbox> `);
+      const input = el.shadowRoot!.querySelector('input')!;
+      expect(input.getAttribute('aria-invalid')).to.equal('false');
+    });
+
+    it('is true when required and checked', async () => {
+      const el = await fixture<HTMLFormElement>(html`<sd-checkbox required checked></sd-checkbox>`);
+      const input = el.shadowRoot!.querySelector('input')!;
```

Comment:
Question: Shouldn't it be a valid input if it is required and checked? 🤔

Reply:
Thanks for catching this